core/src/main/java/io/temporal/samples/earlyreturn/EarlyReturnClient.java

```diff
@@ -77,6 +77,7 @@ private static void runWorkflowWithUpdateWithStart(WorkflowClient client) {
               + ")");
     } catch (Exception e) {
```

Comment:
in description you mention specific grpc exception (StatusRuntimeException)
here just general Exception is asumed its of that type. could add some type check and maybe show how different exceptions could be handled differently in client

Reply:
@tsurdilo I have provided a check for StatusRuntimeException and "ExecuteMultiOperation API is disabled" specifically (with advice). Please let me know if this is enough.